cmd/run_test.go

```diff
@@ -53,3 +53,31 @@ func TestRun(t *testing.T) {
 		})
 	}
 }
+
+// TestRunWithoutConfig verifies
+// that running TTPs still works without a top-level config file
+func TestRunWithoutConfig(t *testing.T) {
+	testCases := []struct {
```

Comment:
important to be able to do this for certain deployment scenarios - now has a test case